wordpress-mobile/WordPress-Android
# Add max width constraint to the My Site fragment layout

This PR fixes the improved My Site fragment layout on tablets, by adding a max width constraint to it:

|Normal|Empty|
|---|---|
|![normal](https://user-images.githubusercontent.com/830056/108919363-a0d88c80-7611-11eb-901d-18fd8b2445f3.png)|![empty](https://user-images.githubusercontent.com/830056/108919368-a2a25000-7611-11eb-8af9-3a0877799942.png)|

To test:

1. Turn on the Improved My Site flag and restart the app.
1. Login with an account that has no sites.
1. Go to the My Site screen.
1. Notice the empty view.
1. Notice the margins are correct.
1. Add a new site.
1. Notice the newly added site appears in the My Site screen.
1. Notice the margins are correct.

---

PR submission checklist:
